CreateDoubleTypeArrayLengthOfFive.java

```diff
@@ -0,0 +1,26 @@
+public class CreateDoubleTypeArrayLengthOfFive {
+    public static void main(String[] args) {
+        double[] arr = {-15, 0, -11.5, 100, -63.9};
+        double[] PosRev = new double[5];// Positive Reversed array
+        if (arr[0] >= 0) {
+            PosRev[4] = arr[0];
+        }
+        if (arr[1] >= 0) {
+            PosRev[3] = arr[1];
+        }
+        if (arr[2] >= 0) {
+            PosRev[2] = arr[2];
+        }
+        if (arr[3] >= 0) {
+            PosRev[1] = arr[3];
+        }
+        if (arr[4] >= 0) {
+            PosRev[0] = arr[4];
+        }
```

Comment:
This is a wrong solution since you need to put all non-negative numbers side by side, starting from the array end to the left. But in your solution {1,-2, 5, 3, -7} produces {0 ,3 ,5 ,0 ,1}, but it should produce {0 ,0 ,3 ,5 ,1}

Reply:
I was not sure if I understood the condition of the exercise correctly, so I tried to solve it in two ways. You can ignore this one because there is another solution for this exercise, which is too difficult to read. I will see if I can find a better one.

Reply:
This is better than the second version from the readability viewpoint. If the second even worked correctly, anyway I wouldn't consider it as a solution, since it isn't readable at all. So modify this solution according to my explanation.